cloudfoundry-cli/parts/plugins/x-cf.py

```diff
@@ -0,0 +1,8 @@
+from snapcraft.plugins import go
+
+
+class CFPlugin(go.GoPlugin):
+
+    def _local_build(self):
+        self._run(['go', 'install', './{}/...'.format(
+            'github.com/cloudfoundry/cli/main')])
```

Comment:
the bug would be that there is no way to snap a project with a tag and multiple main packages.